CIP-xxxx/README.md

```diff
@@ -0,0 +1,136 @@
+   3. CIP authors would select an unused [CBOR] Tag in the [IANA Registry][IANA], and assign it to their data structure.
+      1. The CIP should add to its "Path to Active" that the selected Tag/s have been successfully registered with [IANA].
+         This allows the CIP to be edited with the final Tag once [IANA] have accepted and published the registration.
+   4. CIP Authors would include in their PR an addition to the Cardano [CBOR Tag Registry](#tag-registry).
+
+### Registration of TAGS with IANA
+
+Registration of the Tag with IANA is the sole responsibility of the CIP author.
+CIPs should not proceed to Active if they define unregistered Tags on data structures.
+This is to prevent abuse of the Tag number space.
+
+### Usage of Tags within a CIP
+
+CIPs, such as metadata CIPs can freely define when and how tags are used with any CBOR data structure.
+For example, a CIP may require that a particular field is always tagged.
+They may also define that a field need not be tagged if it is the typical data structure.
+Only less common or unusual data structures would be tagged in this case.
+A CIP may also define that Tags are not used, and only the canonical encoding.
+
+### Canonical Encoding
+
+When a Tag is defines, its Canonical encoding must also be defined.
+This is to ensure that all data that is tagged is encoded in a uniform manor.
+Even if a CIP does NOT use a tag, it should preferably use the canonical encoding for the data structure.
+This is to prevent fragmentation and confusion amongst compliant encoders and decoders of the various data structures.
+
+If a pre-existing data structure is being tagged, then its most common current encoding should be used as the Canonical encoding.
+It should not be re-defined.
+
+For example `ED25519-BIP32` public keys are commonly encoded as a byte array.
+The array is 32 bytes long with the most significant byte of the key appearing first in the array.
+If a Tag is defined for this public key, it should simply define the Tag.
+Its Canonical encoding MUST follow this common encoding scheme.
+
+If a CIP does not refer to a Tag, nor the Canonical encoding specification for the data structure,
+AND it does not define an alternative encoding.
+Then the application implementing the encoding should assume it is encoded canonically.
+This helps ensure backward compatibility with pre-existing CIPs where Tags are not used.
+
+### Tag Registry
+
+Similar to [CIP-0010] this CIP defines a registry of all known tags.
+The format of the registry is defined by the json schema: [CIP Tag Registry Schema].
+New entries should be added to the [CIP Tag Registry] in a PR for a CIP that first defines a new CBOR Tag.
+They MUST be updated when the Tag is accepted or rejected by IANA.
+The registry clearly notes if the tag is currently known to be registered or not.
+If a Tag is not yet registered then any implementor must be aware that its possible the Tag number could change and is not final.
```

Comment:
```suggestion
### Tag Registry

Similar to [CIP-0010] this CIP defines a registry of all known tags.
The format of the registry is defined by the json schema: [CIP Tag Registry Schema].
New entries MUST be added to the [CIP Tag Registry] in a PR for a CIP that first defines a new CBOR Tag.
They MUST be updated when the Tag is accepted or rejected by IANA.
The registry clearly notes if the tag is currently known to be registered or not.
If a Tag is not yet registered then any implementor must be aware that its possible the Tag number could change and is not final.
```
I love that this CIP takes steps to define how to add new tags via a separate CIP. I think I would personally change the "should update registry when adding a new CIP" to a MUST as suggested above. This is very similar, although more brief, to the language we added to CIP-68 for similar extensions. Well done @stevenj on this one!